src/integrationtests/resources/application.properties

```diff
@@ -57,72 +52,58 @@ spring.autoconfigure.exclude=org.springframework.boot.autoconfigure.mongo.embedd
+# Name of the database collection where we store pair of 'subscription:aggregated object'
+# where notification has been sent and should not be repeated
+subscriptions.repeat.handler.collection.name: subscriptions_repeat_handler
+# The waitlist contains Eiffel events which are waiting to be aggregated
+# Configure the database collection name for the waitlist
 waitlist.collection.name: wait_list
-# time to live value in database in seconds for unprocessed events
-waitlist.collection.ttlValue: 600
-# resend information for unprocessed events in wait list, time in miliseconds 
-waitlist.initialDelayResend: 2000
-waitlist.fixedRateResend: 10
-# name of the collection where subscriptions are stored
-subscription.collection.name: subscription
-# name of the collection where we store pair of subscription:aggregated 
-# object where notification has been done and should not be repeated
-subscription.collection.repeatFlagHandlerName: subscription_repeat_handler
-
-# Need to be set to True to use Test Rules page in EI-Frontend.
-# This is when you want to define new aggregation rules and test it
-# on an existing or artificially created set of events.
-testaggregated.enabled: false
-
-# threading settings
-threads.corePoolSize: 100
-threads.queueCapacity: 5000
-threads.maxPoolSize: 150
-
-# WaitList-worker and Scheduling task executor thread settings
-scheduled.threadpool.size: 100
-
-# name of the collection where failed notifications are stored
-failed.notification.collection-name: failed_notification
-# number of retries if a notification fails
-notification.failAttempt: 3
-# time to live in database in seconds for failed notifications
-notification.ttl.value: 600
-
-# email sender settings for email notifications
+# Set the time-to-live value for Eiffel events in the waitlist database (seconds)
+waitlist.collection.ttl: 600
+# Resend information for unprocessed events in waitlist, time in milliseconds
+waitlist.resend.initial.delay: 2000
+waitlist.resend.fixed.rate: 15000
+# Name of the collection where failed notifications are stored
+failed.notifications.collection.name: failed_notifications
+# Configure time-to-live for failed notifications in database (seconds)
+failed.notifications.collection.ttl: 600
+# Number of retries if a notification fail
+notification.retry: 3
+
+# Configure email sender and default subject for email notifications.
 email.sender: noreply@ericsson.com
 email.subject: Email Subscription Notification
 
-# email server settings to route email notifications
-# through
+# email server settings to route email notifications through
 spring.mail.host: localhost
 spring.mail.port: 1025
 spring.mail.username:
 spring.mail.password:
 spring.mail.properties.mail.smtp.auth: false
 spring.mail.properties.mail.smtp.starttls.enable: false
 
-# end point for downstream and upstream search in event repository
+# Url for downstream and upstream search in event repository.
 er.url: http://localhost:8084/search/
```

Comment:
My suggestion is to rename er.url to: event-repository.url

Reply:
I need some more feedback on this one. @e-pettersson-ericsson @a-breid-ericsson What do you think? Should I extend the name on this one?

Reply:
er is a bad name.

Reply:
It is now event.repository.url

Reply:
ok